tools/pubsys/src/aws/ssm/ssm.rs

```diff
@@ -239,7 +239,12 @@ pub(crate) async fn set_parameters(
                 // RusotoError::Unknown(BufferedHttpResponse {status: 400, body: "{\"__type\":\"ThrottlingException\",\"message\":\"Rate exceeded\"}", headers: ...})
                 // Even if we were to do a structural match, we would still have to string match
                 // the body of the error.  Simpler to match the string form.
-                if e.to_string().contains("ThrottlingException") {
+                let error_type = e
+                    .into_service_error()
+                    .code()
```

Comment:
I tried for 30 minutes to find the enum that is providing this string message without success. I guess the stringly typing is the best we can do given how sprawling the library is.

Reply:
From what I can tell, Throttling Exceptions are not currently surfaced: https://github.com/aws/aws-sdk/issues/362